### PR TITLE
Updates the ember uploader, which fixes pdf file type issues, which

### PR DESCRIPTION
**Pull request by variousred**
fixes pdf viewing in chrome

shows validation errors for redirects

related: https://github.com/g5search/g5-content-management-system/pull/851

### CODE REVIEW

**Comment by variousred** [comment]
@levibrown @dreuss @thescubageek

File: app/components/s3-upload.coffee [hunk 1]
```diff
@@ -8,10 +8,9 @@ S3UploadComponent = Ember.FileField.extend
   # S3 will return XML with url
   # => http://yourbucket.s3.amazonaws.com/file.png
   # Uploader will send a sign request then upload to S3
-  filesDidChange: (->
-    uploadUrl = @get("url") + "?locationName=" + @get('websiteName')
-    files = @get("files")
-    uploader = Ember.S3Uploader.create(url: uploadUrl)
+  filesDidChange: (files)->
+    uploadUrl = "http://localhost:3000" + @get("url") + "?locationName=" + @get('websiteName')
```
Comment (thescubageek) [inline app/components/s3-upload.coffee]: Let's get rid of that localhost:3000

File: app/controllers/website/assets.coffee [hunk 2]
```diff
@@ -27,7 +27,34 @@ AssetsController = Ember.ArrayController.extend
 
     deleteAsset: (asset) ->
       website = @get('website')
-      uploader = Ember.S3Uploader.create(url: '/api/v1/sign_delete?locationName=' + website.get('name'))
+      uploader = EmberUploader.S3Uploader.create
+        url: 'http://localhost:3000/api/v1/sign_delete?locationName=' + website.get('name')
```
Comment (thescubageek) [inline app/controllers/website/assets.coffee]: another localhost:3000 to remove

**Comment by variousred** [comment]
@levibrown @thescubageek

**Comment by thescubageek** [comment]
@mmitchellg5 Looks good, is there a CMS with this on it we can take a look at?

**Comment by variousred** [comment]
@thescubageek i deployed it to qatestbeds single domain for byron to check out, he accepted the story

**Comment by thescubageek** [comment]
:+1: good to merge and :ship:

**Pull request merged**